### ISSUE TITLE
Cannot add an instance from a patch that has no parent

### ISSUE THREAD

**Issue by foxfabi** [rojo-rbx/rojo#422]
When I export an object from Roblox Studio into the Rojo file structure, the connection to the Rojo server crashes. 
However, the Rojo server itself in Visual Studio Code continues to run and does not show any messages.

I get the following message in Roblox Studio from the Rojo plugin:

```lua
user_rojo.rbxm.Rojo.Plugin.Reconciler.applyPatch:55: Rojo detected an invariant violation within itself:
Cannot add an instance from a patch that has no parent.
Instance .... with parent ... .
State: InstanceMap {
}
This is a bug in Rojo. Please file an issue...
```

Roblox Studio Plugin: Rojo 6.1.0
Roblox Studio Version: 0.478.5.422443 (64bit) / Latest version available: 0.478.5.422443
Visual Studio Code Version: 1.56.0

May be related to #315

**Comment by glinesbdev** [comment]
Just wanted to also say that I got a similar bug:

```
[Rojo-Warn] Disconnected from an error: cloud_4048317704.Rojo.Plugin.Reconciler.applyPatch:55: Rojo detected an invariant violation within itself:
[Rojo-Warn] Cannot add an instance from a patch that has no parent.
[Rojo-Warn] Instance dad5a5dc-f94a-40ca-93e2-8d77d19629a6 with parent d9ca7219-23bf-428b-b8e1-2a68f89c5970.
[Rojo-Warn] State: InstanceMap {
[Rojo-Warn] }
[Rojo-Warn] 
[Rojo-Warn] This is a bug in Rojo. Please file an issue:
```

Roblox Studio Plugin: Rojo 6.1.0
Roblox Studio Version 0.480.5.423489 (Mac M1 Arm) / Latest version available 0.480.5.423489
Version: 1.56.2

**Comment by Dekkonot** [comment]
If anyone experiences this issue in the future, please open a new one so that we can track it better.
